clueless/views.py

```diff
@@ -60,3 +141,194 @@ def playerturn(request):
 	return HttpResponse(template.render(context,request))
 
 # Controller functions will go below here
+def start_game_controller(request):
+	"""
+	Creates a game with the given host defined by his/her user_id. Also provided
+	is the host's designated character. Other players may join later, as this
+	game is not available in the lobby as a joinable game.
+	"""
+
+	if request.method == 'POST':
+		if'user_id' or 'character_name' not in request.POST:
+			logger.error('user_id or character_name not provided')
+			# TODO Redirect to appropriate error page
+		else:
+			# Gets our expected fields from the user's POST
+			user_id = request.POST('user_id')
+			character_name = request.POST('character_name')
+
+			try:
+				user = User.objects.get(id = user_id)
+			except ObjectDoesNotExist: # Possible User.DoesNotExist
+				logger.info('user not found, adding them now')
+				# TODO create actual user object
+				# TODO add user to database
+
+			try:
+				character = Character.objects.get(name = character_name)
+			except ObjectDoesNotExist: # Possible User.DoesNotExist
+				logger.error('''character not found (Did you forget to add the
+				character in the admin panel?''')
+				# TODO Redirect to appropriate error page
+
+			# Create a Player object for the Host
+			player = Player()
+			player.user = user
+			player.character = character
+
+			# Constructs our game, saves the changes and starts it
+			game = Game()
+			game.addPlayer(player)
+			game.save() # TODO: Check if this is to do be done before startGame()
+			game.startGame()
+	else:
```

Comment:
So I thought we decided logic wise that there would be options to start game or create game, but this method seems to automatically just start a game??